### DISCUSSION TITLE
Disscuss the workflow used by a repo.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [retorquere/zotero-better-bibtex#2759] [Q&A] [answered]
I noticed that there are [several workfolows](https://github.com/retorquere/zotero-better-bibtex/tree/master/.github/workflows) used by this repo, and it seems that most of them work smoothly.

In my case, I want to sync the lyx mirror with the upstream repository with the help of a workflow [here](https://github.com/hongyi-zhao/lyx/blob/master/.github/workflows/sync-with-upstream-repository.yml), but it always failed, see [here](https://github.com/hongyi-zhao/lyx/actions/runs/7596012327/job/20689234971) for more details, as shown below:

![image](https://github.com/ad-m/github-push-action/assets/11155854/6227e270-91c1-496b-8049-8feaef044dd0)

How do you think I could fix this error?

Regards,
Zhao

**Comment by retorquere** [comment]
I don't know, I've not done something like this before, but maybe https://github.com/marketplace/actions/sync-upstream works for you?

**Reply by hongyi-zhao** [reply]
Thank you for letting me know about this one. But based on the below description, it seems that it doesn't work in my case:

![image](https://github.com/retorquere/zotero-better-bibtex/assets/11155854/c584da3e-3e5b-4243-9b15-c0f57bb27826)

**Reply by retorquere** [reply, marked as answer]
Then I don't know, sorry.
